includes/modules/pages/checkout_confirmation/jscript_paypal_incontext.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Javascript enabler for providing and handling the PayPal 
+ *   Express In Context shopping experience.
+ *
+ * @package page
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: Author: mc12345678  Sat Aug 13 11:35:35 2016 -0400 Modified in v1.6.0 $
+ */
+ 
+  if (   defined('MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE') && MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE == 'InContext' 
+      && defined('MODULE_PAYMENT_PAYPALWPP_MERCHANTID')    && MODULE_PAYMENT_PAYPALWPP_MERCHANTID    != '' 
+      && defined('MODULE_PAYMENT_PAYPALWPP_STATUS')        && MODULE_PAYMENT_PAYPALWPP_STATUS        == 'True') {
+        
+        require(DIR_WS_MODULES . 'payment/paypal/paypalwpp_incontext.php');
+?>
+<!-- merchant site HTML page --><script async="async" src="//www.paypalobjects.com/api/checkout.js"></script>
```

Comment:
Comment to be removed and incorporated by forced push again, unless desired to "build" on this "one" at a time? :/ (Or at least to make the same change to all of the files at once instead of something like 6 times over.) Either way will be something I will get to in the next day or two based on accessibility to the desired software/hardware combination. Please continue to provide comments, insightful.